content/guides/core/automations/automation-events.md

```diff
@@ -29,13 +29,48 @@ You can create a project automation in these scopes:
 - The project and all of its collections (the default).
 - All collections that match the filter you supply.
 
-### Events
-These events can trigger a project automation:
+### Artifact events
+This section describes the events related to an artifact that can trigger an automation.
 
 - **Linking a new artifact**: Test and validate new models, datasets or dataset automatically.
 - **Creating a new version of an artifact**: Apply recurring actions to each version of an artifact. For example, start a training job when a new dataset artifact version is created.
 - **Adding a new alias to a version of an artifact**: Trigger a special step of your workflow when a new artifact version in a project or collection has a specific label or alias applied. For example, run a series of downstream processing steps when an artifact has the `test-set-quality-check` alias applied.
 
+### Run metrics events
+This section describes the run metrics events that can trigger an automation.
+
+- **Run metrics threshold met**: Trigger a workflow when, for a given metric, a run or the average of a number of runs meets the threshold you specify for a given metric.
+- **Run metrics change threshold met**: Trigger a workflow when, for a given metric, a run or the average of a number of runs increases or decreases by the threshold you specify for a given metric.
+
+To set up the metric for a run metrics event, you specify:
+
+- An optional run name filter. Only runs matching this filter can trigger the automation.
+- The metric.
+- The threshold for evaluation.
+- The comparison to make. Choices differ for each type of event. Refer to [Run metrics comparison choices]({{< relref "#run-metrics-comparison-choices" >}}).
+
+#### Run metrics comparison choices
+When configuring an automation for a run metric event, you can configure how to compare the run metric value with the threshold you specify. Your choices depend on the event type.
```

Comment:
So this section has a tricky job: it needs to explain how the change threshold metric works. This isn't that complicated, but there are a lot of ways that this *could* be done, and we need to make sure we're very explicit about how it works so that users 

I think it's worth separating out the two types of run metrics (threshold vs change threshold) and explaining the algorithms separately, specifically what values are included in the windows (eg that the current & prior don't overlap and relative ordering), and how the windows are compared (for the change threshold)

Let me know if you'd like to meet to discuss the details of this.

Reply:
I think this is done, PTAL!